README.md

```diff
@@ -145,8 +145,9 @@ laptop asset should be converted to a folder, and the license moved into it.
   created, and template files generated and committed.
 
   The current working directory will be initialized if neither `directory` nor
-  the `ONYO_REPOSITORY_DIR` environment variable are specified. If both are set,
-  Onyo will throw an error.
+  the `ONYO_REPOSITORY_DIR` environment variable are specified. If the
+  environment variable is set, but a directory variable is given, Onyo will use
+  the given directory.
```

Comment:
I disagree with this change in behavior (what if we ad a `-C` option like `git`; how should that behave?). But, if you do plan to make this change, the wording should be made more explicit:

> If the both are set, Onyo will print a warning, but still initialize the supplied directory.

Reply:
I reverted this change. The code will be updated later. I will also create an issue for -C, where we can discuss the implementation and the behaviour of it.